robottelo/ui/contentviews.py

```diff
@@ -212,8 +212,8 @@ def check_progress_bar_status(self, version):
         while check_progress and time.time() <= timer:
             check_progress = self.wait_until_element(
                 (strategy, value % version),
-                timeout=6,
-                poll_frequency=2,
+                timeout=1,
+                poll_frequency=0.5,
```

Comment:
That new type of alert ('alert.success_sub_form') is disappearing in 1.5-2 seconds, if we set timeout to 6 for progress bar we can miss the alert. Setting it to 1 sec as we are waiting for progress bar to disappear in a loop anyways.